registrations/permissions.py

```diff
@@ -88,4 +116,57 @@ def has_object_permission(
         if request.method == "DELETE":
             return obj.can_be_deleted_by(request.user)
 
+        if request.method in ("PUT", "PATCH"):
+            return self._has_object_update_permission(request, obj)
+
+        return obj.can_be_edited_by(request.user)
+
+
+class CanAccessSignupGroup(SignUpBasePermission):
+    @staticmethod
+    def _has_object_update_permission(request: Request, obj: SignUpGroup) -> bool:
+        if request.user.is_superuser or request.user.is_registration_admin_of(
+            obj.publisher
+        ):
+            return True
+
+        def get_signups_keys():
+            keys = set()
+            for signup in request.data.get("signups", []):
+                keys |= set(signup.keys())
+            return keys
+
+        if obj.registration.registration_user_accesses.filter(
+            email=request.user.email
+        ).exists():
+            # Registration user can only change presence_status and nothing else,
+            # and they need to be strongly identified.
+            data_keys = get_signups_keys()
+            return request.user.is_strongly_identificated and (
```

Comment:
is_strongly_identificated was renames as is_strongly_identified in https://github.com/City-of-Helsinki/linkedevents/pull/715